kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/ColumnMapping.java

```diff
@@ -93,6 +100,68 @@ public static StructType convertToPhysicalSchema(
         }
     }
 
+    /** Returns the physical name for a given {@link StructField} */
+    public static String getPhysicalName(StructField field) {
+        if (field.getMetadata().contains(COLUMN_MAPPING_PHYSICAL_NAME_KEY)) {
```

Comment:
can we use the utility method `hasPhysicalName` here?

Reply:
good point, done